Common/include/CConfig.hpp

```diff
@@ -6686,6 +6688,20 @@ class CConfig {
    */
   const su2double* GetInlet_Velocity(string val_index) const;
 
+  /*!
+   * \brief Get the mass fraction vector for a NEMO supersonic inlet boundary.
+   * \param[in] val_index - Index corresponding to the inlet boundary.
+   * \return The inlet velocity vector.
+   */
+  const su2double* GetInlet_MassFrac() const { return Inlet_MassFrac; }
+
+  /*!
+   * \brief Get the Tve value for a NEMO supersonic inlet boundary.
```

Comment:
```suggestion
   * \brief Get the Tve value for a NEMO inlet boundary.
```

Reply:
```suggestion
   * \brief Get the Tve value for a NEMO supersonic inlet boundary.
```

Just in case we use these for subsonic inlets.